hmp/simulations.py

```diff
@@ -346,21 +348,20 @@ def demo(cpus, n_events, seed=123):
     """Create example data for the tutorials."""
     ## Imports and code specific to the simulation (see tutorial 3 and 4 for real data)
     from scipy.stats import gamma
-
-    from hmp.utils import read_mne_data
+    from hmp.io import read_mne_data
 
     random_gen = np.random.default_rng(seed=seed)
 
     ## Parameters for the simulations
     frequency, amplitude = (
         10.0,
-        0.3e-7,
-    )  # Frequency of the transition event and its amplitude in Volt
+        1e-7,
```

Comment:
These magic numbers should at some point be defined somewhere, but not for this PR.